Source/JavaScriptCore/inspector/remote/cocoa/RemoteInspectorCocoa.mm

```diff
@@ -125,71 +123,95 @@ static bool globalAutomaticInspectionState()
 {
 }
 
+void RemoteInspector::initialize()
+{
+    ASSERT(isMainThread());
+    if (!m_pendingMainThreadInitialization)
```

Comment:
This access to `m_pendingMainThreadInitialization` does not have the `Lock`, but all other accesses do.

You could perhaps just make the field an `Atomic<bool>`?